### PR TITLE
Getting started guide: master --> end

### PR DESCRIPTION
**Pull request by tjvantoll**
This PR changes the getting started guide to say that the completed code of the guide lives in Groceries' “end” branch rather than “master”, as we need the ability to evolve master without screwing up the state of this tutorial.

This is a small change but I could use a quick review of the wording. Ping @jlooper @ikoevska

### CODE REVIEW

**Comment by jlooper** [comment]
I believe it's ok.

File: getting-started/chapter-4.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ exports.signIn = function() {
 };
 ```
 
-> **TIP**: You can always view the completed codebase in the 'master' branch of the [sample-Groceries repo](https://github.com/NativeScript/sample-Groceries.git).
+> **TIP**: You can always view the completed codebase in the “master” branch of the [sample-Groceries repo](https://github.com/NativeScript/sample-Groceries/tree/end).
```
Comment (ikoevska) [inline getting-started/chapter-4.md]: I'm getting some conflicting information here :) Is it the master or the end branch?

**Comment by ikoevska** [comment]
Fix the master/end controversy in chapter 4 and you're good to go :)

File: getting-started/chapter-4.md (hunk 1 above)
Comment (tjvantoll) [inline getting-started/chapter-4.md]: Wow. And this is why proofreading is important :)

Thanks for catching this. I'll fix this and merge the changes in.

**Comment by ligaz** [comment]
Should we change the default branch to be the `end` one?

**Comment by tjvantoll** [comment]
> Should we change the default branch to be the end one?

Well this is tough, because Groceries is also the first app listed on https://www.nativescript.org/app-samples-with-code, so we want to make sure that what we have in the default branch has all the features we want to show off.

IMO we could either 1) put a big warning on the top of the README that points to the end branch, or 2) put the getting started guide code in its own repo.

**Pull request merged**